### PR TITLE
Updated font, border and background colors of disabled styles

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
The problem is we often disable inputs while data is saving data to the server, so this new disabled style is more jarring now.

https://switchcomm.atlassian.net/browse/DT-168

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
 - [x] Request a review from Brad Paugh, David Becher, or Drew Chandler.

### CODE REVIEW

**Comment by dialpad-drew** [comment]
Something I didn't catch in the design: can we make these arrows on the right the same color as the text?

Screen Shot 2021-10-06 at 4 46 06 PM

**Comment by juliodialpad** [comment]
In order to get that done, I'll need to add a class to dialtone **d-select--disabled**, it's ok if I do it this way? 

It's hard to style the arrow based on disabled state because the disabled state is on a children of d-select and the :before and :after that represent the arrows are siblings of .d-select__input.

image

**Comment by dialpad-drew** [comment]
Hmmm, [CSS Tricks](https://css-tricks.com/almanac/selectors/d/disabled/) says you can chain these, though I haven't been able to get it to work.

> :disabled can be “chained” with other pseudo-selectors: like :before or :after to generate icons or text to provide more user feedback.

**Comment by juliodialpad** [comment]
> Hmmm, [CSS Tricks](https://css-tricks.com/almanac/selectors/d/disabled/) says you can chain these, though I haven't been able to get it to work.
> 
> > :disabled can be “chained” with other pseudo-selectors: like :before or :after to generate icons or text to provide more user feedback.

The real problem here is that you cannot style a parent element based on child status, I've been investigating and seems the only way to do that is javascript, or adding a class like I did

**Comment by dialpad-drew** [comment]
I'm ok w/ this. I'd like to hold off and get @braddialpad 's 👍  or 👎 before merging. Also, we'll need to update the Vue component as well with this new class, so I'll make a story for that.

**Comment by dialpad-drew** [comment]
> > Hmmm, [CSS Tricks](https://css-tricks.com/almanac/selectors/d/disabled/) says you can chain these, though I haven't been able to get it to work.
> > > :disabled can be “chained” with other pseudo-selectors: like :before or :after to generate icons or text to provide more user feedback.
> 
> The real problem here is that you cannot style a parent element based on child status, I've been investigating and seems the only way to do that is javascript, or adding a class like I did

BTW, is the class you added pushed up? I don't see the style update in the deploy preview.

**Comment by juliodialpad** [comment]
> > > Hmmm, [CSS Tricks](https://css-tricks.com/almanac/selectors/d/disabled/) says you can chain these, though I haven't been able to get it to work.
> > > > :disabled can be “chained” with other pseudo-selectors: like :before or :after to generate icons or text to provide more user feedback.
> > 
> > 
> > The real problem here is that you cannot style a parent element based on child status, I've been investigating and seems the only way to do that is javascript, or adding a class like I did
> 
> BTW, is the class you added pushed up? I don't see the style update in the deploy preview.

I was waiting for comments to push it, I'll push it so you can see it.

**Review by dialpad-drew — APPROVED** [review]
LGTM, but want to wait for Brad.

File: docs/components/select.html [hunk 1]
```diff
@@ -85,7 +85,7 @@
   <div class="d-label">
     <label for="Dialtone--SelectExample2">...</label>
   </div>
-  <div class="d-select">
+  <div class="d-select d-select--disabled>
```
Comment (braddialpad) [inline docs/components/select.html]: Missed a quote here.

**Comment by braddialpad** [comment]
> In order to get that done, I'll need to add a class to dialtone **d-select--disabled**, it's ok if I do it this way? 
 

yep this is fine IMO

**Review by braddialpad — CHANGES_REQUESTED** [review]

File: docs/components/select.html (hunk 1 above)
Comment (juliodialpad) [reply docs/components/select.html]: Fixed, thanks

**Review by braddialpad — APPROVED** [review]

**Pull request merged**